Code/Framework/AzCore/AzCore/Asset/AssetSerializer.h

```diff
@@ -197,7 +197,7 @@ namespace AZ {
 
         static ClassInfoType* GetGenericInfo()
         {
-            return GetCurrentSerializeContextModule().CreateGenericClassInfo<ThisType>();
+            return GetGlobalSerializeContextModule().CreateGenericClassInfo<ThisType>();
```

Comment:
`GetCurrentSerializeContextModule` was renamed to `GetGlobalSerializeContextModule` because its meaning is no longer relevant. When AzCore was static, each module had managed its own collection of module contexts, but now that its a shared library, there is only one global collection for all the modules that are loaded.